transport_nantes/press/views.py

```diff
@@ -296,3 +297,25 @@ def get_image_extension(file_contents):
         press_mention.og_image_url.save(image_filename, files.File(image_fp))
         press_mention.save()
         return None
+
+
+@csrf_protect
+@permission_required('press.press-editor')
+def check_for_duplicate(request, *args, **kwargs):
+    """Check if the provided URL already exists in PressMention."""
+    url = request.GET.get("url")
+    if url:
+        press_mention = PressMention.objects.filter(article_link=url).first()
```

Comment:
It's not about being unique, it's about having an object or None (instead of 404) 
The article_link has already UNIQUE constraint, we can't have more than 1 result.

I'll add a comment about this.

Reply:
It's still better for your code to express what you mean than to have a comment that explains that the code expresses something else.